omegaconf/errors.py

```diff
@@ -81,6 +81,12 @@ class InterpolationToMissingValueError(InterpolationResolutionError):
     """
 
 
+class InterpolationValidationError(InterpolationResolutionError):
```

Comment:
make it also extend ValidationError.

Reply:
Done in 6159f3e
(whose commit message has the wrong exception name -- oops -- but shouldn't be an issue since we squash PRs)